src/api/posts/src/data/post.js

```diff
@@ -35,6 +35,28 @@ function ensureFeed(feed) {
   return feed instanceof Feed ? Promise.resolve(feed) : Feed.byId(feed);
 }
 
+/**
+ * Determine the type of the post depending on the given URL.
+ * The possible types are 'video' or 'blogpost'.
+ *
+ * @param {*} url
+ * @returns a String representing the type
+ */
+function determinePostType(url) {
+  try {
+    const associatedLink = new URL(url);
+
+    if (associatedLink.hostname.includes('youtube.com')) {
```

Comment:
There are a bunch of formats for YouTube domains, see https://gist.github.com/rodrigoborgesdeoliveira/987683cfbfcc8d800192da1e73adc486.  I wonder how many we might hit in the wild via the feed API?

Reply:
How should we approach this? Do we write an entire list of YouTube domains to check against?

Reply:
I'm not sure. Let's start with something simple (what you have here), and remember that this might be something we have to fix later on.  I suspect that YouTube will normalize the URLs in the feeds.